.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: test
+
+on: [push]
```

Comment:
Note that this won't trigger for pull releases that initiate from external repositories. You need `pull_request_target` for that. See https://github.com/visgl/loaders.gl/pull/1016 and https://github.com/visgl/loaders.gl/pull/1018